graylog2-web-interface/src/components/lookup-tables/CacheTableEntry.tsx

```diff
@@ -80,16 +80,16 @@ const CacheTableEntry = ({ cache }: Props) => {
     return `${NumberUtils.formatNumber(hitRate)}%`;
   };
 
-  const handleEdit = (cacheName: string) => () => {
-    history.push(Routes.SYSTEM.LOOKUPTABLES.CACHES.edit(cacheName));
+  const handleEdit = () => {
```

Comment:
Can you wrap this with a `useCallback`? This way the function is not regenerated on each render.